sklearnex/svm/nusvr.py

```diff
@@ -65,6 +65,17 @@ def __init__(
     def fit(self, X, y, sample_weight=None):
         if sklearn_check_version("1.2"):
             self._validate_params()
+        elif self.nu <= 0 or self.nu > 1:
```

Comment:
I would prefer to add this check to the dispatch level, to the onedal_cpu/gpu_support functions, to faalback stock sklearn let him deal with error raising